src/dev/src/app/content-container.component.ts

```diff
@@ -56,5 +58,20 @@ import { APP_ROUTES } from './app.routing';
         `,
 })
 export class AppContentContainerComponent {
+  private static readonly ROOT_BREADCRUMB_ELEMENT = { label: 'Home', url: '/' };
+  private static readonly TEST_BREADCRUMB_ELEMENT = { label: 'breadcrumb', url: 'breadcrumb' };
+
   public routes: Route[] = APP_ROUTES;
+
+  constructor(private breadcrumbService: ClrBreadcrumbService, private router: Router) {
+    router.events.subscribe(val => {
```

Comment:
You still got a tslint error here (can be seen at pull request detail page at the bottom), telling that you don't use the router variable declared in constructor.
Because constructor parameters always get declared in "this" object you need to type "this.router.events...." for the linter to get green.